frameworks/elastic/tests/config.py

```diff
@@ -112,7 +112,7 @@ def wait_for_expected_nodes_to_exist(service_name=SERVICE_NAME, task_count=DEFAU
     stop_max_delay=DEFAULT_ELASTIC_TIMEOUT*1000,
     retry_on_result=lambda res: not res)
 def check_kibana_plugin_installed(plugin_name, service_name=SERVICE_NAME):
-    cmd = "bash -c '$MESOS_SANDBOX/kibana-$ELASTIC_VERSION-linux-x86_64/bin/kibana-plugin list'"
+    cmd = "KIBANA_DIRECTORY=$(ls -d $MESOS_SANDBOX/kibana-*-linux-x86_64); $KIBANA_DIRECTORY/bin/kibana-plugin list"
```

Comment:
The `;` is not strictly required, right?

Reply:
It is in this case, same issue as in the comment below.